Dockerfile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+RUN pip install --upgrade pip
+
+RUN apt-get update
+
+RUN apt-get install curl unzip --yes
+
+RUN apt-get update && apt-get install -y nodejs npm
+RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && apt-get install -y nodejs
+
+RUN python3 -m venv /opt/venv
+
+RUN . /opt/venv/bin/activate
+
+RUN pip install pynecone-io
+
+WORKDIR /pynecone
+
+RUN mkdir demo_pynecone_app
+
+WORKDIR /pynecone/demo_pynecone_app
+
+RUN pc init
+
+CMD ["pc","run"]
+
+EXPOSE 3000
+EXPOSE 8000
```

Comment:
Why do you expose port 8000?

Reply:
Port 8000 is the backend FastAPI port, while the NextJS app is on port 3000. So we need the 8000 port to connect to the websocket